app/controllers/providers/accreditations/check_controller.rb

```diff
@@ -0,0 +1,75 @@
+module Providers
+  module Accreditations
+    class CheckController < ::CheckController
+      before_action :load_provider
+
+    private
+
+      def load_provider
+        @provider = policy_scope(Provider).find_by!(uuid: params[:provider_id])
```

Comment:
how come this is not using the new `find` from `app/models/concerns/uuid_identifiable.rb`